crates/bevy_ecs/src/query/fetch.rs

```diff
@@ -1607,6 +1608,24 @@ macro_rules! impl_anytuple_fetch {
 all_tuples!(impl_tuple_fetch, 0, 15, F, S);
 all_tuples!(impl_anytuple_fetch, 0, 15, F, S);
 
+/// WorldQuery used to make operations no-ops by turning `Query<Q>` into `Query<NopWorldQuery<Q>>`
```

Comment:
```suggestion
/// [`WorldQuery`] used to nullify queries by turning `Query<Q>` into `Query<NopWorldQuery<Q>>`
```
I try to avoid literal repetition between names and docs.